lms/envs/production.py

```diff
@@ -307,6 +307,9 @@
 # Mobile store URL overrides
 MOBILE_STORE_URLS = ENV_TOKENS.get('MOBILE_STORE_URLS', MOBILE_STORE_URLS)
 
+# Learner Profile URL overrides
+PROFILE_MICROFRONTEND_URL_BASE = ENV_TOKENS.get('PROFILE_MICROFRONTEND_URL_BASE', PROFILE_MICROFRONTEND_URL_BASE)
```

Comment:
Do you have an related PRs to wire up this override?

Reply:
Here they are: 
https://github.com/edx/configuration/pull/5018
https://github.com/edx/edx-internal/pull/731